test.sh

```diff
@@ -1,7 +1,23 @@
 #!/bin/bash
-echo "Checking gofmt..."
-files=$(gofmt -l -s ./tests)
-if [[ $files ]]; then
-    echo "Gofmt errors in files: $files"
-    exit 1
+set -e
+set -x
+export TAG=$2
+export REPO=$1
+export VERSION=$3
+export BUCKET=$4
+GCLOUD_CMD="gcloud"
+if [ -n "$5" ]
+then
+  GCLOUD_CMD=$5
 fi
+
+if [ "$VERSION" == "jessie" ]
```

Comment:
It's to support different versions in the future as they get rolled out. We need to support as many as three versions at a time. We'll get rid of EOLed versions from the if statement as they're deprecated.